### DISCUSSION TITLE
Question: How to disable _hover styles globally for mobile

### DISCUSSION THREAD

**Discussion by AdlerJS** [chakra-ui/chakra-ui#4167] [Q&A]
The Problem: `_hover` can cause issues with UI such as sometimes forcing the user to double tab the item when on mobile to activate a click event. Also, just in general _hover doesn't really make sense to have as a concept in a mobile touch environment anyways. I am wondering if there is a way to disable the _hover globally for mobile devices so we don't have to locate all instances of the _hover pesudo selector in the app and use ternaries to determine the style.

I.e This doesn't seem scalable or fun to maintain

```  
 _hover={{
        bg: isMobile ? null : 'gray.100',
      }}
```

**Comment by with-heart** [comment]
> `_hover` can cause issues with UI such as sometimes forcing the user to double tab the item when on mobile to activate a click event

That shouldn't ever happen. If you could open a bug report issue with a reproduction of that situation, we can definitely investigate that.

**Reply by AdlerJS** [reply]
@with-heart I'll work on getting a demo put together that can be reproducible. In the meantime is there a way to disable _hover globally and conditionality. Or do we need to handle that on a case by case basis?
